src/styles/index.scss

```diff
@@ -2,2 +1,134 @@
+@import './variables';
+
+.page-body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
The `justify-content: center;` property is not necessary here unless you expect the content to not take up the full width of the `.card` and want to center it. If the content takes up the full width, you can safely remove this property.